### ISSUE TITLE
Add "Number of items answered" metric in the overview table

### ISSUE THREAD

**Issue by zkoppert** [github/issue-metrics#312]
### Is your feature request related to a problem?

When running the action on discussions, it would be nice to be able to see the "Number of items answered" metric in the overview table

### Describe the solution you'd like

If `discussion` is found in the `search_query` then count the number of discussions marked as answered and display the count in the markdown and json output.

**Comment by mark-mxwl** [comment]
Hi @zkoppert! I've been reviewing this issue, and would be happy to help out. Before diving in, I'd like to clarify a few things to make sure I fully understand the requirements for the feature:

1. After reviewing the discussion query schema, I assume that the field to include in the condition is `answerChosenAt`. Ex.: If the value is not `None`, add to the count.
2. I assume that the logic would be implemented in `issue_metrics.py`, and the following modules would need to be updated to account for the changes: `markdown_writer.py`, `json_writer.py`, and any relevant test scripts.
3. Is there a particular position in the table where you'd like `Number of items answered:` to appear?

Any thoughts, suggestions, or guidance you could offer would be greatly appreciated!

**Comment by Piedone** [comment]
As a user of this action, I'd comment on the 3rd question: I'd add it after "Number of items that remain open", i.e. here:

![Image](https://github.com/user-attachments/assets/aa6ccfba-d01c-4345-af20-0a6d411f1600)

For an example of the current output on an active project see: https://github.com/OrchardCMS/OrchardCore/issues/17442.

**Comment by mark-mxwl** [comment]
Thanks, @Piedone! Appreciate your input, as well as the sample output. 👌

**Comment by zkoppert** [comment]
@mark-mxwl yup that all sounds correct to me and I agree with @Piedone on the placement.

**Comment by mark-mxwl** [comment]
Thanks for confirming, @zkoppert! I'll ping you when I have a PR up. ⚡️
